### PR TITLE
Type fixes, closes #155

### LINKED ISSUE #155: Type signatures causing errors in build process
Since I've made dependency updates im facing errors caused by type definition file.

Installed version(s):

````
    "@editorjs/checklist": "^1.6.0",
    "@editorjs/code": "^2.9.0",
    "@editorjs/editorjs": "^2.30.5",
    "@editorjs/header": "^2.8.7",
    "@editorjs/list": "^1.10.0",
    "@editorjs/marker": "^1.4.0",
    "@editorjs/quote": "^2.7.2",
    "@editorjs/table": "^2.4.1",
    "editorjs-tooltip": "^1.2.2",
````

sc

This line within the file `node_modules/@editorjs/table/dist/toolbox.d.ts:29:38` causing the error: 

Bildschirmfoto 2024-08-28 um 23 10 53

and modifying it like this solved it.

Bildschirmfoto 2024-08-28 um 23 11 12

### PR DESCRIPTION
**Pull request by Sozialarchiv**
This ticket fixes the type erros, mentioned in https://github.com/editor-js/table/issues/155.

Resolves #155

### CODE REVIEW

**Review by neSpecc — CHANGES_REQUESTED** [review]
Please, attach an issue correctly and write a nice description

File: src/toolbox.js [hunk 1]
```diff
@@ -14,11 +24,12 @@ export default class Toolbox {
   /**
    * Creates toolbox buttons and toolbox menus
    *
-   * @param {object} api - Editor.js api
-   * @param {PopoverItem[]} items - Editor.js api
-   * @param {function} onOpen - callback fired when the Popover is opening
-   * @param {function} onClose - callback fired when the Popover is closing
-   * @param {string} [cssModifier] - the modifier for the Toolbox. Allows to add some specific styles.
+   * @param {Object} config
+   * @param {any} config.api - Editor.js api
```
Comment (neSpecc) [inline src/toolbox.js:28]: could you import types form editor js package?

Comment (Sozialarchiv) [reply src/toolbox.js:28]: The jsdoc is just a workaround. If you really want to good types the best solution is to migrate to ts.

Comment (neSpecc) [reply src/toolbox.js:28]: I mean something like this

```js
/**
 * @typedef {import('./types').MyType} MyType
 */
```

Comment (neSpecc) [reply src/toolbox.js:28]: Hey, @Sozialarchiv would you be able to try it?

Comment (Sozialarchiv) [reply src/toolbox.js:28]: I will take a look at it.

**Review by neSpecc — APPROVED** [review]

**Pull request merged**